website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -41,12 +41,16 @@
         <br />
         <br />
         <div class="row" data-bind="if: showSelect">
-          <div class="col-md-8">
+          <div class="col-md-6">
```

Comment:
It didn't have one before. But this also isn't a true form; there is simply a button data bound to another function on the view model using the values data bound to the `select`